connman/plugins/sailfish_ofono.c

```diff
@@ -582,13 +599,102 @@ static int modem_configure(struct modem_data *md)
 	return index;
 }
 
+static gboolean is_modem_network_configured(struct modem_data *md)
+{
+	DBG("%p", md);
+
+	switch (md->connctx->protocol) {
+	case OFONO_CONNCTX_PROTOCOL_UNKNOWN:
+	case OFONO_CONNCTX_PROTOCOL_NONE:
+		return FALSE;
+	case OFONO_CONNCTX_PROTOCOL_IP:
+		if (!connman_network_is_configured(md->network,
+						CONNMAN_IPCONFIG_TYPE_IPV4)) {
+			connman_warn("ofono: %p IPv4 no address set", md);
+			return FALSE;
+		}
+
```

Comment:
Quite a few empty lines in this function could be deleted to make it more compact. This file is not unstreamable and therefore can afford to be more readable than the rest of the connman code)) Feel free to ignore though.

Reply:
Added some and I guess I got your drift here.